commands/unequip.js

```diff
@@ -0,0 +1,129 @@
+const funcall = require("../modules/funcall.js");
+//simple ping command to check if the bot is online.
+const strifecall = require("../modules/strifecall.js");
+exports.type = "sylladex";
+exports.desc = "Unequip a weapon or other equipment"
+exports.use = `";unequip" removes your currently-equipped weapon from your specibus and puts it in your sylladex.
```

Comment:
change ; to >, the help menu auto-adjusts > to whatever the designated prefix is.